test/test_resume.cpp

```diff
@@ -77,7 +79,7 @@ std::shared_ptr<torrent_info> generate_torrent()
 	for (piece_index_t i(0); i < fs.end_piece(); ++i)
 	{
 		sha1_hash ph;
-		for (int k = 0; k < 20; ++k) ph[k] = lt::random(0xff);
+		for (std::size_t k = 0; k < 20; ++k) ph[k] = lt::random(0xff) & 0xff;
```

Comment:
what do you think of: ``for (auto& v : ph) v = lt::random(0xff) & 0xff);``?

Reply:
or just `aux::random_bytes(ph);` :)